daemon/cmd/policy_test.go

```diff
@@ -699,7 +699,7 @@ func (ds *DaemonSuite) TestReplacePolicy(c *C) {
 			EgressCommonRule: api.EgressCommonRule{
 				ToCIDR: []api.CIDR{
 					"1.1.1.1/32",
```

Comment:
Should we also then change the CF addrs? Those addrs are used as dummy, i.e. no actual traffic is sent, so I don't see a big problem with both.